src/Internals/WCCom/WCConnectionHelper.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Helpers for managing connection to WooCommerce.com.
+ *
+ * @package Automattic\WooCommerce\Internals\WCCom
+ */
+
+namespace Automattic\WooCommerce\Internals\WCCom;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class WCConnectionHelper.
+ *
+ * Helpers for managing connection to WooCommerce.com.
+ */
+final class WCConnectionHelper {
```

Comment:
Since it uses `WCCom` namespace, we could just use `ConnectionHelper`

Reply:
Renamed in https://github.com/woocommerce/woocommerce/pull/26932/commits/4500e86198729ce893945a39fe9e22cee3991856 . Although I have kept the same name by using the keyword `as` in WCInstall class because I felt just ConnectionHelper class is pretty generic and can conflict with something in future.